Index: server/src/main/java/com/launchcode/RecLeagueOrganizer/RecLeagueOrganizerApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.launchcode.RecLeagueOrganizer;\r\n\r\nimport org.springframework.boot.SpringApplication;\r\nimport org.springframework.boot.autoconfigure.SpringBootApplication;\r\n\r\n@SpringBootApplication\r\npublic class RecLeagueOrganizerApplication {\r\n\r\n\tpublic static void main(String[] args) {\r\n\t\tSpringApplication.run(RecLeagueOrganizerApplication.class, args);\r\n\t}\r\n\r\n}\r\n
===================================================================
--- server/src/main/java/com/launchcode/RecLeagueOrganizer/RecLeagueOrganizerApplication.java	(revision deba664b5f2ef98658e540efcae41c2918f33979)
+++ server/src/main/java/com/launchcode/RecLeagueOrganizer/controllers/RecLeagueOrganizerApplication.java	(date 1607394973460)
@@ -1,4 +1,4 @@
-package com.launchcode.RecLeagueOrganizer;
+package com.launchcode.RecLeagueOrganizer.controllers;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
Index: server/src/main/resources/templates/fragments.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- server/src/main/resources/templates/fragments.html	(date 1607395682098)
+++ server/src/main/resources/templates/fragments.html	(date 1607395682098)
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
\ No newline at end of file
